tests/test_scripts/test_packages_exporter.py

```diff
@@ -1,24 +1,21 @@
 import os
 import tempfile
 
-import pytest
-from syncer import sync
-
+from alws.models import SignKey
 from scripts.packages_exporter import Exporter
+from tests.mock_classes import BaseAsyncTestCase
 
 IN_GITHUB_ACTIONS = os.getenv("GITHUB_ACTIONS") is not None
 
 
-@pytest.mark.skip(
-    reason="See https://github.com/AlmaLinux/build-system/issues/204"
-)
-def test_repomd_signer():
-    exporter = Exporter(
-        pulp_client=None, repodata_cache_dir='~/.cache/pulp_exporter'
-    )
-    db_keys = sync(exporter.get_sign_keys())
-    key_id = db_keys[0]['keyid']
-    with tempfile.NamedTemporaryFile() as fp:
-        fp.write(b'Hello world!')
-        res = sync(exporter.sign_repomd_xml(fp.name, key_id))
-        assert res['error'] is None
+class TestPackagesExporter(BaseAsyncTestCase):
+    async def test_repomd_signer(self, sign_key: SignKey):
+        exporter = Exporter(
+            pulp_client=None, repodata_cache_dir='~/.cache/pulp_exporter'
+        )
+        key_id = sign_key.keyid
+        token = await exporter.get_sign_server_token()
+        with tempfile.NamedTemporaryFile() as fp:
```

Comment:
instead of using the `tempfile` module, you can use the built-in `tmp_path` fixture
https://docs.pytest.org/en/stable/how-to/tmp_path.html

Reply:
fixed